### PR TITLE
Update checkpoint paths and add demo index page.

### PR DESCRIPTION
**Pull request by adarob**

### CODE REVIEW

File: music/demos/build-demos.sh [hunk 1]
```diff
@@ -26,7 +26,7 @@
 if [[ -n $1 ]]; then
   demos=( "${1}" )
 else
-  demos=( *.html )
+  demos=( *.ts )
```
Comment (cghawthorne) [inline music/demos/build-demos.sh]: My thought for using .html was that we might eventually have demos with multiple .ts files, but they'll probably be launched from one html file. But I see the problem with index.html now. We could either specifically exclude it or just go with this approach until we need something more complicated. :-)

File: music/demos/music_rnn.ts [hunk 2]
```diff
@@ -151,7 +152,8 @@ async function runMelodyRnn() {
   console.log(tf.memory());
 }
 
-async function runDrumsRnn() {
+async function
+runDrumsRnn() {
```
Comment (cghawthorne) [inline music/demos/music_rnn.ts]: why is this on 2 lines?

File: music/demos/music_vae.ts [hunk 3]
```diff
@@ -293,8 +294,8 @@ runTrio() {
 }
 
 // TODO(adarob): Switch to magenta/core function once implemented.
-function concatNoteSequences(seqs: mm.INoteSequence[],
-    individualDuration: number) {
+function
+concatNoteSequences(seqs: mm.INoteSequence[], individualDuration: number) {
```
Comment (cghawthorne) [inline music/demos/music_vae.ts]: ditto

Comment (adarob) [reply music/demos/music_vae.ts]: Done.

File: music/demos/music_rnn.ts (hunk 2 above)
Comment (adarob) [reply music/demos/music_rnn.ts]: Done.

**Pull request merged**